src/ralph/discovery/http.py

```diff
@@ -38,11 +54,12 @@ def get_http_info(ip):
         logger.error("Exception: {} for url: {}".format(e, url))
         url = "https://{}".format(':'.join((ip, '8006')))  # for Proxmox
         try:
-            response = requests.get(url, timeout=REQUEST_TIMEOUT, verify=False)
+            response = http_get_method(session, url)
             response.raise_for_status()
             headers, content = response.headers, response.content
         except requests.exceptions.RequestException as e:
             logger.error("Exception: {} for url: {}".format(e, url))
+    session.close()
```

Comment:
maybe better than finally block will be with statement around whole ``try..execpt`` blocks?